doc/releases/release-notes-2.4.rst

```diff
@@ -303,6 +319,8 @@ Drivers and Sensors
 
   * STM32: Number of changes including k_malloc removal, driver piority init
     increase, get_status API addition and various cleanups.
+  * Added MCUX EDMA driver for i.MX RT and Kinetis K6x SoCs.
+  * Added MCUX LPC driver for LPC and i.MX RT6xx SoCs.
```

Comment:
The word DMA is missing. Add MCUX LPC "DMA" driver

Reply:
I didn't include it because this is a sub-bullet in the DMA drivers section